lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -490,11 +490,12 @@ class EditingState {
 /// This corresponds to Flutter's [TextInputConfiguration].
 class InputConfiguration {
   InputConfiguration({
-    required this.inputType,
-    required this.inputAction,
-    required this.obscureText,
-    required this.autocorrect,
-    required this.textCapitalization,
+    this.inputType = EngineInputType.text,
```

Comment:
`required` also implies non-null. After this change, this constructor will allow `null` values for fields marked as nullable. It might be a good time to revisit the nullness of the fields that are being assigned by this constructor. In particular, I see that when parsing the configuration from `flutterInputConfiguration` below, we are not assigning the same defaults as we do here.

Reply:
Good call about the different in default. I'll fix it.